code/modules/antagonists/slaughter/slaughter.dm

```diff
@@ -57,6 +64,23 @@
 	if(istype(loc, /obj/effect/dummy/phased_mob/slaughter))
 		bloodspell.phased = TRUE
 
+/mob/living/simple_animal/slaughter/CtrlShiftClickOn(atom/A)
```

Comment:
a once per 45 second yeet is nice, I guess, but how the hell is a slaughter demon player who doesn't codedive know that they can do this?

Reply:
save this slaughter demon stuff for another PR

Reply:
No. I'll add a chat message about the slam but I drastically lowered the bleeding from brute damage so they need the extra bleeding wounds to compensate